source/daplink/drag-n-drop/vfs_user.c

```diff
@@ -111,6 +111,15 @@ void vfs_user_build_filesystem()
         file_size = get_file_size(read_file_need_bl_txt);
         vfs_create_file("NEED_BL TXT", read_file_need_bl_txt, 0, file_size);
     }
+    
+#ifdef BOARD_VFS_ADD_FILES    
```

Comment:
I would recommend using a weak reference for board_vfs_add_files rather than a define.